plotly/offline/offline.py

```diff
@@ -9,9 +9,11 @@
 import os
 import uuid
 import warnings
-from pkg_resources import resource_string
 import webbrowser
 
+from pkg_resources import resource_string
+
+
```

Comment:
it looks like this got accidentally committed - GitHub Desktop makes it easy to not commit and discard these types of changes